homeassistant/components/ipp/config_flow.py

```diff
@@ -111,10 +119,16 @@ async def async_step_zeroconf(self, discovery_info: ConfigType) -> Dict[str, Any
         except IPPConnectionUpgradeRequired:
             return self.async_abort(reason="connection_upgrade")
         except (IPPConnectionError, IPPResponseError):
+            _LOGGER.debug("IPP Connection/Response Error", exc_info=True)
             return self.async_abort(reason="connection_error")
         except IPPParseError:
-            _LOGGER.exception("IPP Parse Error")
+            _LOGGER.debug("IPP Parse Error", exc_info=True)
             return self.async_abort(reason="parse_error")
+        except IPPVersionNotSupportedError:
+            return self.async_abort(reason="ipp_version_error")
+        except IPPError:
+            _LOGGER.debug("IPP Error", exc_info=True)
+            return self.async_abort(reason="ipp_error")
```

Comment:
Just an observation that the try... except blocks in the user and discovery steps are very similar. There's just two things that doesn't match. Maybe possible to consolidate somehow in the future?